app/Http/Middleware/TrustProxies.php

```diff
@@ -1,25 +0,0 @@
-<?php
```

Comment:
No no no no no.

This file must stay or your are breaking the IP transfer for users running Lychee on VM with proxy.
Or you have another solution.

This one is build breaking for me.

Reply:
Also see this: https://github.com/LycheeOrg/Lychee/issues/1197

Reply:
I see. Will add the trusted proxy middleware again.

Reply:
I added the middleware `TrustProxies` again, but directly call the parent class now. There is absolutely no need to extend a class if it is not extended.

Reply:
True, but the point of `TrustProxies` is to overload this part:
```php
	protected $proxies = [
		'10.0.2.2',
	];
```
Otherwise the parent class is useless, or am I failing to understand your goals.

I am considering having this array to be instantiated via a `.env` and `split(' ')`

Reply:
The idea of the parent class is to load those values from a config file. The config file has been added to this PR. So there is absolutely no need to extend the class.